ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+ProductNotification.swift

```diff
@@ -24,87 +24,32 @@ extension BrowserViewController {
-      let internalURL = InternalURL(url)
-    {
-      isAboutHomeUrl = internalURL.isAboutHomeURL
-    }
-
-    guard let selectedTab = tabManager.selectedTab,
-      !Preferences.General.onboardingAdblockPopoverShown.value,
-      !benchmarkNotificationPresented,
-      !Preferences.AppState.backgroundedCleanly.value,
-      Preferences.Onboarding.isNewRetentionUser.value == true,
-      !topToolbar.inOverlayMode,
-      !isTabTrayActive,
-      selectedTab.webView?.scrollView.isDragging == false,
-      isAboutHomeUrl == false
-    else {
-      return
-    }
-
-    let blockedRequestURLs = selectedTab.contentBlocker.blockedRequests
-
-    if !blockedRequestURLs.isEmpty, let url = selectedTab.url {
-
-      let domain = url.baseDomain ?? url.host ?? url.schemelessAbsoluteString
-
-      guard currentBenchmarkWebsite != domain else {
-        return
+      var isAboutHomeUrl = false
+      if let url = tabManager.selectedTab?.url, let internalURL = InternalURL(url) {
+        isAboutHomeUrl = internalURL.isAboutHomeURL
       }
 
-      currentBenchmarkWebsite = domain
-
-      guard
-        let trackersDetail = BlockedTrackerParser.parse(
-          blockedRequestURLs: blockedRequestURLs,
-          selectedTabURL: url
-        )
+      // The conditions guarding presentation for educational notifications
+      guard let selectedTab = tabManager.selectedTab,
+        presentedViewController == nil,
+        !isOnboardingOrFullScreenCalloutPresented,
+        !Preferences.AppState.backgroundedCleanly.value,
+        !topToolbar.inOverlayMode,
+        !isTabTrayActive,
+        selectedTab.webView?.scrollView.isDragging == false,
+        isAboutHomeUrl == false
       else {
         return
       }
 
-      notifyTrackersBlocked(
-        domain: domain,
-        displayTrackers: trackersDetail.displayTrackers,
-        trackerCount: trackersDetail.trackerCount
-      )
-
-      Preferences.General.onboardingAdblockPopoverShown.value = true
-
+      self.presentEducationalProductNotifications(selectedTab: selectedTab)
     }
   }
 
-  private func presentEducationalProductNotifications() {
-    if Preferences.DebugFlag.skipEduPopups == true { return }
-
-    var isAboutHomeUrl = false
-    if let selectedTab = tabManager.selectedTab,
-      let url = selectedTab.url,
-      let internalURL = InternalURL(url)
-    {
-      isAboutHomeUrl = internalURL.isAboutHomeURL
-    }
-
-    guard let selectedTab = tabManager.selectedTab,
-      presentedViewController == nil,
-      !benchmarkNotificationPresented,
-      !isOnboardingOrFullScreenCalloutPresented,
-      !Preferences.AppState.backgroundedCleanly.value,
-      !topToolbar.inOverlayMode,
-      !isTabTrayActive,
-      selectedTab.webView?.scrollView.isDragging == false,
-      isAboutHomeUrl == false
+  private func presentEducationalProductNotifications(selectedTab: Tab) {
+    // Data Saved Pop-Over only exist in JP locale
+    guard Locale.current.regionCode == "JP",
```

Comment:
You can remove the locale check it is used in clause in the function or remove the one in if clause